src/main/java/eu/darkbot/api/entities/Pet.java

```diff
@@ -1,6 +1,74 @@
 package eu.darkbot.api.entities;
 
+import eu.darkbot.utils.ThrowableItem;
+
 public interface Pet extends Ship {
+
     int getLevel();
     int getOwnerId();
+
+    enum Buff {
```

Comment:
```suggestion
    enum Cooldown {
```
While in-game they may be considered buffs, i think we should call them Cooldowns and remove the "_CD" suffix that a few of those have